py-polars/polars/functions.py

```diff
@@ -14,15 +15,44 @@
 
 from typing import Dict
 from .datatypes import DataType
+from urllib.parse import (  # noqa
+    urlencode,
+    urljoin,
+    urlparse as parse_url,
+    uses_netloc,
+    uses_params,
+    uses_relative,
+)
+
+_VALID_URLS = set(uses_relative + uses_netloc + uses_params)
+_VALID_URLS.discard("")
 
 
 def _process_http_file(path: str) -> io.BytesIO:
     with urllib.request.urlopen(path) as f:
         return io.BytesIO(f.read())
 
 
+def _is_url(url):
```

Comment:
Could you add the types in the function description encapsulated by quotation marks (this is needed for python 3.6 support)

Reply:
Modified, please review. Thank you for your guidance.